src/commcare_cloud/fab/fabfile.py

```diff
@@ -192,6 +192,7 @@ def env_common():
         'staticfiles': staticfiles,
         'staticfiles_primary': [staticfiles[0]],
         'lb': [],
+        # deploy contains one server to which commcare-hq is deployed
```

Comment:
Took me a couple reads to understand the comment as worded here. I struggled to make it clearer on first read, but this is what I came up with to suggest.

```suggestion
        # 'deploy' contains a single server, one that commcare-hq is deployed to.
```